### DISCUSSION TITLE
Comments' scroll button enhancement

### DISCUSSION THREAD

**Discussion by armync** [Docile-Alligator/Infinity-For-Reddit#965] [Ideas]
**Description**: The scroll button for the comments in the curent state is a bit wasteful. Infinity client has rather a _chunky_ one which also is unmovable. On the other hand, the offical Reddit client has it easy on use, _smaller_, _movable_ _everywhere_ on the screen when pressed ~and _permanently available_  (in comparison to the Infinity's disappearance at scroll)~.

**Comparison**:

**Infinity**

![Screenshot_20220221-212343_Infinity](https://user-images.githubusercontent.com/16375271/155016936-0d558b5e-35af-45d3-aff3-b861e4c7370f.jpg)

**Official Reddit**

![Screenshot_20220221-212229_Reddit](https://user-images.githubusercontent.com/16375271/155017010-594993a3-92ce-4404-bc27-7d2bdcc58361.jpg)

**PS:** Is there any way to disable the _Comment save_ button? I tend to press it quite often by mistake.

### Thanks for the work!

**Comment by Docile-Alligator** [comment]
For now, you can enable Lock Jump to Next Top-level Comment Button in Settings->Gestures & Buttons.
To hide the comment save button, you can chage Settings->Interface->Comment->Show Fewer Toolbar Options Starting From to Level 0.
